cli/cmd/helm/helmInstall.go

```diff
@@ -37,13 +37,19 @@ func addHelmInstallFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolP("enable-timescaledb-backup", "b", false, "Enable TimescaleDB S3 backup")
 	cmd.Flags().StringP("timescaledb-tls-cert", "", "", "Option to provide your own tls certificate for TimescaleDB")
 	cmd.Flags().StringP("timescaledb-tls-key", "", "", "Option to provide your own tls key for TimescaleDB")
+	cmd.Flags().StringP("version", "", "", "Option to provide your tobs helm chart version, if not provided will install the latest tobs chart available")
+	cmd.Flags().BoolP("only-secrets", "", false, "Option to create only TimescaleDB secrets")
```

Comment:
These two options should really have been a separate PR or commit. Ok for now, but lets be more careful in future

Reply:
`only-secrets` I agree. But `version` is added as a utility for upgrade tests. We use `--version` to install `0.2.2` and to upgrade to the latest. So added it as a part of this PR.